ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -194,47 +209,49 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{nam
                                         part + ' '
                                     )
                                 );
-
+                            const {Tree, Pods, Network, List} = AppsDetailsViewKey;
                             return (
                                 <div className='application-details'>
                                     <Page
-                                        title='Application Details'
+                                        title={this.getPageTitle(pref.view) + ' - ' + this.props.match.params.name}
```

Comment:
With multiple open tabs application name won't fit and user will see it. Can you reverse the order please?

Reply:
Ok, will reverse it to:

`title={this.props.match.params.name + ' - ' + this.getPageTitle(pref.view)}`